support-frontend/assets/pages/supporter-plus-landing/supporterPlusLanding.tsx

```diff
@@ -298,7 +272,22 @@ export function SupporterPlusLandingPage({
 								>
 									<AmountAndBenefits />
 								</Box>
-								<div css={displayContributionsStripeCss(stripeDisplayed)}>
+								{optimisedMobileLayout2 && (
+									<StickyCta
```

Comment:
Can see moved to exactly where its displayed on screen as suggested would be best